v3.1/glfw/util.go

```diff
@@ -1,11 +1,36 @@
 package glfw
 
+//#include <stdlib.h>
 //#include "glfw/include/GLFW/glfw3.h"
 import "C"
 
+import (
+	"reflect"
+	"unsafe"
+)
+
 func glfwbool(b C.int) bool {
 	if b == C.GL_TRUE {
 		return true
 	}
 	return false
 }
+
+func bytes(origin []byte) (pointer *uint8, free func()) {
+	if len(origin) == 0 {
+		panic("Slice of bytes cannot be empty.")
+	}
+
+	l := len(origin)
```

Comment:
Nitpick, but I like `n` better than `l` here. It's more consistent with most similar Go code I've seen.